build.sbt

```diff
@@ -267,52 +116,25 @@ lazy val commonSettings = Seq(
 )
 
 lazy val commonJvmSettings = Seq(
-  testOptions in Test += Tests.Argument(TestFrameworks.ScalaTest, "-oDF")
+  Test / testOptions += Tests.Argument(TestFrameworks.ScalaTest, "-oDF")
 )
 
 lazy val commonJsSettings = Seq(
-  scalaJSStage in Global := FastOptStage,
+  Global / scalaJSStage := FastOptStage,
   parallelExecution := false,
   jsEnv := new org.scalajs.jsenv.nodejs.NodeJSEnv(),
   coverageEnabled := false
 )
```

Comment:
basically none of these settings are needed, except for coverage.

Reply:
there are only four right? So you are saying "in commonJsSettings you may remove the first three settings"

The "basically" is confusing me since it sounds like you are saying something imprecise by adding that word.

Reply:
Yes you caught me :) To be precise:
- `Global / scalaJSStage := FastOptStage` is the default and unneeded
- `jsEnv := new org.scalajs.jsenv.nodejs.NodeJSEnv()` is the default and unneeded
- `parallelExecution := false` is not the default, but I'd be surprised if it's actually needed
- `coverageEnabled := false` is also quite likely not needed, there is coverage on SJS but its occasionally buggy